### ISSUE TITLE
Improved Look

### ISSUE THREAD

**Issue by velitasali** [trebleshot/trebleshot.github.io#7]
Having a webpage was a good start, and it already brings visitors to the project.

As a second attempt to it, I suggest improving the webpage by making things look less cluttered and more in line with the design.

## Initial suggestions

* Use up-to-date screenshots.
* Fix alignment issues. The title is a bit out of place.
* Use more shadows.
* Use lighter colors.
* Make the project statistics dynamic (stars and issues will reflect the real numbers) or remove them altogether.
* Mention the desktop project.
* The logo on the main page is too large. You can change it to a screenshot that could be more of a vector image than an actual representation. 

You can resolve these issues one by one by opening multiple pull requests.

Also, please leave a comment here if you are going to work on any of these issues.

**Comment by reveurguy** [comment]
can i work on the shadows and colors part?

**Comment by velitasali** [comment]
@reveurguy Of course :)

**Comment by reveurguy** [comment]
ok, i will start working on it. can i connect with you on any other platform so that i can show you the changes before making a PR and asking your feedback also.

**Comment by velitasali** [comment]
@reveurguy Yes. Also, you can open the pull request at any time. You don't have to wait until you are finished.

**Comment by reveurguy** [comment]
where can i connect? yeah but i think that i will make the pr when finished and till then show you the changes on any other social platform so that i don't have to make many PR's before the final one.

**Comment by velitasali** [comment]
@reveurguy I am mostly on Telegram at **t.me/velitasali**.

**Comment by reveurguy** [comment]
are you on instagram or discord?

**Comment by velitasali** [comment]
I have a discord account **tasali**#0861

**Comment by reveurguy** [comment]
i have sent you a friend req there

**Comment by devarshidoshi** [comment]
Can u assign me the shadow and colors work if open??

**Comment by velitasali** [comment]
@devarshidoshi yes, no one is working on that task.

**Comment by devarshidoshi** [comment]
Thankk Youu I have did the necessary changes in your sites color theme according to your logo and have also added shadow effect
You can preview it here https://devarshidoshi.github.io/trebleshot.github.io/ If you need any other change u can tell me or else if you love it i will send you pull request

PS: can you please also add hacktoberfest topic in your issue

**Comment by devarshidoshi** [comment]
@velitasali waiting for your feedback...

**Comment by velitasali** [comment]
@devarshidoshi Looks nice! You can create a pull request so that we can discuss it over there.

**Comment by devarshidoshi** [comment]
Thanks For Compliments @velitasali had created a PR can you please change hacktober spelling in repository topic
